src/data/resolvers/colonyActions.ts

```diff
@@ -264,9 +295,7 @@ export const colonyActionsResolvers = ({
           isWhitelistActivated: false,
           verifiedAddresses: [],
           colonySafes: [],
-          safeData: null,
-          safeTransactions: [],
-          transactionsTitle: '',
+          safeTransactionStatuses,
```

Comment:
This should default to empty array like one below?

Reply:
The default is already an empty array in a few lines above: `let safeTransactionStatuses: TRANSACTION_STATUS[] = [];`